sycl/include/CL/sycl/detail/sycl_mem_obj_t.hpp

```diff
@@ -81,6 +83,7 @@ class SYCLMemObjT : public SYCLMemObjI {
 
   virtual ~SYCLMemObjT() = default;
 
+  const plugin &getPlugin() const { return MInteropContext->getPlugin(); }
```

Comment:
Do you use the method? If no I suggest removing it.
The problem here is that MInteropContext can be/usually it's nullptr. It's not null when user creates a memory object using interop constructor only.

Reply:
I cannot remove it, since it is used in sycl_mem_obj_t.hpp. 
I have added an assert check on the pointer.